fardf: init location

Noki opened the pull request:
This PR is a first draft for converting fardf to bbb-configs based on the current setup, not following the bbb-configs conventions. The main motivation for this PR is to solve the instability issues with the current Falter firmware running at the core.

TODO:

- [x] Get a new IPv6 prefix and configure it
- [x] Get a new IPv4 prefix that could be used, split it and use it for MGMT during migration
- [x] Check if there are any special config settings that need to be preserved (found only outdated stuff)
- [x] Prepare SNMP and Airos DFS reset configuration
- [x] Create additional VLANs at the switch
- [x] Have at least one person review this
  - [x] Check prepared config and compare it to the existing setup; Are there any issues?

The migration would be as follows:

- [x] Switch all antennas to the new management VLAN and ip-addresses including dns, gateway and NTP
- [x] Change the switch to use the new management VLAN and ip-addresses including dns, gateway and NTP
- [x] Build and flash a minimal image for the core and flash it <= ~7 MB so partition resize could take place
- [x] Place pwd file into root directory
- [x] Reconfigure antennas to use the new Mesh VLANs
- [x] Remove VLAN 101 from configuration
- [x] Build a normal image for the core and flash it so we get full functionality
- [x] Remove old VLANS from switch

After bbb-configs migration:

- [x] Update UISP
- [x] Unregister old IP prefixes
- [x] Update Wiki

pmelange commented:
For the migration, it is necessary to do all the VLAN changes manually before the first mini-image flash.  Otherwise you will be locked out.

pmelange commented:
mesh interface "rudi" is not needed.  The device doesn't exist.  history: there was an attempt to make the connection to a garden, but it didn't work.

pmelange commented:
the contact info can be set to "community" and not specifically just me.

pmelange commented:
mesh interface BHT can be set to mesh mode "ether" as it's a client of the dish at BHT.

Noki commented:
Thanks for the feedback.

> mesh interface "rudi" is not needed. The device doesn't exist. history: there was an attempt to make the connection to a garden, but it didn't work.

removed it

> the contact info can be set to "community" and not specifically just me.

changed it

> mesh interface BHT can be set to mesh mode "ether" as it's a client of the dish at BHT.

set it to p2p

> For the migration, it is necessary to do all the VLAN changes manually before the first mini-image flash. Otherwise you will be locked out.

Hmm, I thought having the VLANs like I do would be a drop in replacement... So just to clarify:

Instead I should do:

- add missing VLANs to the switch (done)
- reconfigure each antenna to use the new VLAN 42 with the new IP/DNS/Gateway/NTP server for management
- reconfigure the switch to use the new VLAN 42 with the new IP/DNS/Gateway/NTP server for management
- change the bbb-configuration to directly reflect those changes
- flash the image

right?

btw,; the north antenna seems to be down and therefore can't be reconfigured.

pmelange commented:
> Hmm, I thought having the VLANs like I do would be a drop in replacement... So just to clarify:
> 
> Instead I should do:
> 
>     * add missing VLANs to the switch (done)
> 
>     * reconfigure each antenna to use the new VLAN 42 with the new IP/DNS/Gateway/NTP server for management
> 
>     * reconfigure the switch to use the new VLAN 42 with the new IP/DNS/Gateway/NTP server for management
> 
>     * change the bbb-configuration to directly reflect those changes
> 
>     * flash the image
> 
> 
> right?

I would leave the migration to you. I'm sure it'll work however you do it.  In the past, I and @PolynomialDivision migrated everything manually with the running image until it was ready for the bbbconfig image.  But like the saying goes, there's more than one way to skin a cat.

pmelange commented:
> btw,; the north antenna seems to be down and therefore can't be reconfigured.

That antenna is attached via sange (poe passthrough).  I can see it's on and that the link is at 1gbps.  But I also can't reach it.  Maybe someone has to go there and reset it and reconfigure it.

Noki commented:
This PR is ready for review. Maybe someone could find some time to doublecheck if all the VLANs are setup correctly, especially the pass through VLANs.

@pmelange please unregister the old IPs
@FFHener please make sure the wiki-updater updates the wiki based on this config

pmelange commented:
> @pmelange please unregister the old IPs

done
